browser/public/index.html

```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/torch.svg" />
```

Comment:
I think we should have a `favicon.ico` file as well for a fallback for non-supporting browsers, and/or adding the `rel="mask-icon"` for Safari. https://caniuse.com/?search=favicon%20svg

Reply:
Good call, I forgot how much of a headache favicons were. I'll be adding an svg and png backup and aligning it with different rels.